content/_index.de.md

```diff
@@ -1,27 +1,31 @@
-## Was ist Freifunk?
+## {{< icon icon="wifi" >}} Was ist Freifunk?
```

Comment:
This causes issues as this changes the text of the heading to `wifi Was ist Freifunk?` we really don't want that.

![image](https://github.com/user-attachments/assets/664f1e80-4ea7-4a89-b14f-3e64a6e89706)

Reply:
Ah yes, we really don't want that.